app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/address/AddressJourneyType.scala

```diff
@@ -81,6 +81,16 @@ object AddressJourneyType {
       isATrust: Boolean
     ) extends Returns
 
+    final case class EnteringSingleMixedUsePropertyDetails(
+      journey: FillingOutReturn,
+      draftReturn: DraftSingleMixedUseDisposalReturn,
+      representativeType: Option[
```

Comment:
have you merged in the master? This should be `Option[RepresentativeType]` now

Reply:
I replaced all instances where we had `Either[PersonalRepresentative, Capacitor]` wth `RepresentativeType` to accommodate for period of admin